src/app/admin/students/students.component.html

```diff
@@ -46,6 +47,6 @@ <h3>Студенти</h3>
 
   </div>
 
-  <div *ngIf="!STUDENTS_LIST.length" class="noStudents">Студенти відсутні</div>
+  <div *ngIf="STUDENTS_LIST.length == undefined" class="noStudents">Студенти відсутні</div>
```

Comment:
Was trying without .length and it appeared to be not working correctly, found somewhere in a forum that it can be a solution to have it like this

Reply:
I think STUDENTS_LIST.length === 0 should works fine. It case it didn't work root cause of it should be found.
I took a quick glance and didn't find case when length can be undefined. And you shouldn't use == in js because of unpredictable type conversion.
By the way STUDENS_LIST should be renamed to studentList

Reply:
Got it

Reply:
Found the root problem and fixed the issue, not the ngIf statement is STUDENT_LIST === undefined or STUDENT_LIST !== undefined
Fix code was commited and pushed